person_capture/gui_app.py

```
@@ -880,6 +881,130 @@ def _prescan_cache_root(self, cfg: SessionConfig) -> Path:
+                rot = int(rec.get("rot_deg", 0))
+                prefix = f"f{idx:08d}_{phase}_c{cand_idx:02d}_r{rot}_fd{fd:.3f}"
+                if isinstance(chip, np.ndarray) and chip.size > 0:
+                    chips_dir.mkdir(parents=True, exist_ok=True)
+                    chip_name = f"{prefix}_chip.png"
+                    if cv2.imwrite(str(chips_dir / chip_name), chip):
+                        rec["chip_path"] = str(Path("chips") / chip_name)
+                if annotated is not None:
+                    try:
+                        x1, y1, x2, y2 = [int(v) for v in bbox[:4]]
+                        h, w = annotated.shape[:2]
+                        x1 = max(0, min(w - 1, x1)); x2 = max(x1 + 1, min(w, x2))
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)